detectron2/modeling/roi_heads/mask_head.py

```diff
@@ -151,7 +158,7 @@ def mask_rcnn_inference(pred_mask_logits, pred_instances):
         instances.pred_masks = prob  # (1, Hmask, Wmask)
 
 
-class BaseMaskRCNNHead(nn.Module):
+class BaseMaskRCNNHead(nn.Sequential):
```

Comment:
This might have unexpected side effect for other subclasses. It's safer to limit this change to `MaskRCNNConvUpsampleHead` for now.

Reply:
The `forward` function of `MaskRCNNConvUpsampleHead` is defined in `BaseMaskRCNNHead `, you mean make a copy of the `forward` function of `BaseMaskRCNNHead ` and add it to `MaskRCNNConvUpsampleHead`?

Reply:
···
class MaskRCNNConvUpsampleHead(BaseMaskRCNNHead, nn.Sequential):
```
seems to work

Reply:
done:)